packages/apps/shopify-api/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "baseUrl": ".",
     "outDir": ".",
-    "rootDir": "."
+    "rootDir": ".",
+    "useDefineForClassFields": false
```

Comment:
Our tests were failing before adding this.
We can either use this option or update our test code.
https://github.com/microsoft/TypeScript/issues/52331